dev-infrastructure/docs/development-setup.md

```diff
@@ -259,6 +265,13 @@ Option 2: You can follow the below manual steps from the root of the CS repo on
 # Setup the development.yml
 cp ./configs/development.yml .
 
+# Setup the azure-runtime-config.json
+cp ./configs/azure/example-config.json ./azure-runtime-config.json
+
+#  Get azure-first-party-application-client-id
+CLIENT_ID=$(az ad app list --display-name aro-hcp-dev-first-party --query '[*]'.appId -o tsv)
```

Comment:
There are severa lclient IDs so I would be more specific and name the variable FPA_CLIENT_ID